31-temperature-and-smoke-detector/README.md

```diff
@@ -19,72 +22,63 @@ Temperature monitoring and smoke detection across a number of rooms at a facilit
 
 Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
 
-## Sparrow Setup
-
-### Quickstart
-
-Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) pair your Sparrow reference node with the gateway and associate it with the Notehub project you just created. Note also that you only need one reference node for this project (although multiple reference nodes can be used to monitor multiple rooms). Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the [`hub.set`](https://dev.blues.io/api-reference/notecard-api/hub-requests/#hub-set) command in the quickstart.
+## LoRa Gateway Setup
 
-After you've completed the quickstart, leave the Notecarrier and Sparrow Essentials board powered and connected. These two devices will act as our gateway to Notehub, and you won't need to touch them again. The rest of this guide will focus on the Sparrow Reference Sensor node.
+Before you can use the Notecard LoRa you need to have a LoRaWAN gateway that is provisioned to The Things Network.  To make this easy you can use the [Blues Indoor LoRaWAN Gateway](https://shop.blues.com/products/blues-starter-kit-lorawan).  To get this set up follow the [setup instructions](https://dev.blues.io/lora/connecting-to-a-lorawan-gateway/)
 
-### MQ2 Sensor Connection
+## Swan Setup
 
-This solution makes use of the an MQ2 sensor module, which detects smoke, as well as various gasses. Using the 3 male-to-female jumper wires, connect the MQ2 sensor to the Sparrow Reference Sensor board and the v5 converter as follows:
+First connect your Blues Swan and Notecard to your Notecarrier-F.
 
-1. Splice a male to female jumper in half and solder the exposed wires to the `GND` pad on the 5v converter. Connect the female end to `GND` on the sensor and the male end to the `GND` pin on Sparrow.
-2. Splice another jumper, and solder the exposed wire on male half to `Vi` on the converter. Connect the male connector to `VBAT` on Sparrow. Solder the exposed wire on the female half to `Vo` and connect the female end to the `A0` pin on the MQ2 sensor.
-3. Using an intact male-to-female jumper, connect `A0` on the sensor to the `A1` pin on Sparrow.
-4. The 4th pin on the sensor is left unconnected.
+1. Follow the steps in the [Notecard Quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#connect-your-notecard-and-notecarrier) to connect your Notecard LoRa to your Notecarrier-F.  Your antenna will only have one cable, and the Notecard LoRa only has one connection.
```

Comment:
```suggestion
1. Follow the steps in the [Notecard Quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#connect-your-notecard-and-notecarrier) to connect your Notecard LoRa to your Notecarrier F.  Your antenna will only have one cable, and the Notecard LoRa only has one connection.
```

Reply:
There are a few other “Notecarrier-F” references still in this doc so a change all to make them all “Notecarrier F”.